packages/beacon-node/src/eth1/eth1DepositDataTracker.ts

```diff
@@ -67,6 +72,8 @@ export class Eth1DepositDataTracker {
   /** Dynamically adjusted batch size to fetch deposit logs */
   private eth1GetLogsBatchSizeDynamic = MAX_BLOCKS_PER_LOG_QUERY;
   private readonly forcedEth1DataVote: phase0.Eth1Data | null;
+  /** To stop `runAutoUpdate()` in addition to AbortSignal */
+  private stopPolling: boolean = false;
```

Comment:
if a node is started after `EIP6110_FORK_EPOCH`, `runAutoUpdate()` should not run so need to initialize this in the constructor, chain will decide the value of it

Reply:
> if a node is started after `EIP6110_FORK_EPOCH`, `runAutoUpdate()` should not run so need to initialize this in the constructor, chain will decide the value of it

Will properly implement deprecation of eth1 polling in the next PR. Added todo. Thanks